docs/API.md

```diff
@@ -66,6 +66,7 @@ All iOS boolean options can also be specified as `string`
 
 Attribute | Type | Default | Description
 --------- | ---- | ------- | -----------
+`ios.voip` | `boolean` | `false` | Optional. If `true` the device will be set up to receive VoIP Push notifications and the other options will be ignored since VoIP notifications are silent notifications that should be handled in the "notification" event.
```

Comment:
This is a bit unclear. So if `voip` is set to `true` does this mean that you will not be able to receive regular notifications that show up in the notification shade?

Reply:
In the future we could enhance the plugin to be able to handle the two types of notifications, but currently it is the expected behavior.